src/android/CDVBroadcaster.java

```diff
@@ -137,6 +155,24 @@ public void run() {
             callbackContext.success();
             return true;
         }
+        else if (action.equals("sendGlobalBroadcast")) {
+            final String eventName = args.getString(0);
+            if( eventName==null || eventName.isEmpty() ) {
+                callbackContext.error(EVENTNAME_ERROR);
```

Comment:
let's make sure the style matches the rest of the existing code (white-space inside the parens, and such)